src/test/java/duke/command/CommandParserTest.java

```diff
@@ -0,0 +1,174 @@
+package duke.command;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import duke.exception.DateParseException;
+import duke.exception.DukeException;
+import duke.exception.IncompleteTaskException;
+import duke.exception.InvalidTaskException;
+import duke.exception.UnknownCommandException;
+
+public class CommandParserTest {
+    @Test
+    public void parseCommand_addTaskCommand() {
+        CommandParser parser = new CommandParser();
+        try {
+            assertTrue(parser.parseCommand("todo exercise") instanceof AddTaskCommand);
+        } catch (DukeException e) {
+            fail("Exception should not be thrown");
+        }
+    }
+
+    @Test
+    public void parseCommand_incompleteTodoDescription_exceptionThrown() {
```

Comment:
Good use of featureUnderTest_testScenario_expectedBehavior(). I'll have to change my test code naming for this part as i didn't follow this format :L

To improve it further, i think you can consider describing specifically what kind of exceptions are thrown in ...exceptionThrown()